src/routes/_surfaces/workspace/explore/time-controls/time-range-utils.ts

```diff
@@ -415,13 +415,13 @@ const roundDateUp = (date: Date | undefined, timeGrain: TimeGrain): Date => {
       const interval = 24 * 60 * 60 * 1000;
       return new Date(Math.ceil(date.getTime() / interval) * interval);
     }
-    // case TimeGrain.OneWeek: {
-    //   // rounds to the next Monday
-    //   const day = date.getUTCDay();
-    //   const dateRoundedDownByDay = roundDateDown(date, TimeGrain.OneDay);
-    //   const timeUntilMonday = (day === 0 ? 1 : 8 - day) * 24 * 60 * 60 * 1000;
-    //   return new Date(dateRoundedDownByDay.getTime() + timeUntilMonday);
-    // }
+    case TimeGrain.OneWeek: {
+      // rounds to the next Monday
+      const day = date.getUTCDay();
+      const dateRoundedDownByDay = date; //roundDateDown(date, TimeGrain.OneDay);
```

Comment:
@ericpgreen2 do we have an issue without roundDateDown?

Reply:
wasn't sure where it was in the code, was giving me a squiggly

Reply:
Just added it back in to the code